ipfx/bin/run_pipeline_from_nwb_file.py

```diff
@@ -18,16 +18,41 @@ def main():
     parser = argparse.ArgumentParser(
         description="Process an nwb file through the ephys pipeline"
     )
-    parser.add_argument("input_nwb_file", type=str)
-    parser.add_argument("output_dir", type=str)
-    parser.add_argument("--write_spikes",
-                        type=bool,
-                        default=False,
-                        help="If true will attempt to append spike times to the nwb file",
-                        )
-    parser.add_argument("--input_json", type=str, default="input_json")
-    parser.add_argument("--output_json", type=str, default="output_json")
-    parser.add_argument("--qc_fig_dir", type=str, default=None)
+    parser.add_argument(
+        "input_nwb_file", type=str, help="process this NWB2 file"
+    )
+    parser.add_argument(
+        "output_dir", type=str, help="outputs will be written here"
+    )
+    parser.add_argument(
+        "--write_spikes", type=bool, default=False,
+        help="If true will attempt to append spike times to the nwb file",
+    )
+    parser.add_argument(
+        "--input_json", type=str, default="input_json",
+        help=(
+            "write pipeline input json file here (relative to "
+            "OUTPUT_DIR/cell_name, where cell_name is the extensionless "
+            "basename of the input NWB file)"
+        )
+    )
+    parser.add_argument(
+        "--output_json", type=str, default="output_json", 
+        help=(
+            "write output json file here (relative to OUTPUT_DIR/cell_name, "
+            "where cell_name is the extensionless basename of the input NWB "
+            "file)"
+        )
+    )
+    parser.add_argument(
+        "--qc_fig_dir", type=str, default=None, const="qc_figs", nargs="?",
+        help=(
+            "Generate qc figures and store them here (relative to "
+            "OUTPUT_DIR/cell_name, where cell_name is the extensionless "
+            "basename of the input nwb file). If you supply --qc_fig_dir with " 
+            "no arguments, the path will be OUTPUT_DIR/cell_name/qc_figs"
```

Comment:
It is worth mentioning that the default value will make no figures.

Reply:
done